setup/unattended-install/unattended-install.php

```diff
@@ -0,0 +1,314 @@
+<?php
+$bBypassMaintenance = true;
+require_once(dirname(__FILE__, 3) . '/approot.inc.php');
```

Comment:
```suggestion
require_once(dirname(__FILE__, 2) . '/approot.inc.php');
```

Shouldn't it be 2 instead?

Reply:
indeed it does not work with current proposal

Reply:
Just this, then ok for me

Reply:
Discussed IRL with Olivier. He tested in real conditions with "3" and it works.

Reply:
It probably is "2" when you use `__DIR__`.